### ISSUE TITLE
When deleting rc, pod does not get deleted

### ISSUE THREAD

**Issue by runseb** [skippbox/kompose#13]
I need to check how kubectl does this, but usually when you delete a rc, it also deletes the pods that this RC created.
Might be two api calls wrapper in kubectl delete ...

**Comment by runseb** [comment]
yes, so kubectl has a reaper to delete related resources:

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/delete.go#L141

**Comment by ngtuna** [comment]
Yes you're right. `Kubectl` automatically turns on `cascade=true`. I'm figuring out how to add this flag to API request in `Kompose`. If can not do that, I will add api call for deleting the corresponding pod.

**Comment by ngtuna** [comment]
```When using the REST API or go client library, you need to do the steps explicitly (scale replicas to 0, wait for pod deletions, then delete the replication controller).```

**Comment by bgrant0607** [comment]
You might want to wait for K8s 1.3. We're going to support server-side cascading deletion.

https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/garbage-collection.md

cc @janetkuo

**Comment by runseb** [comment]
@bgrant0607 thanks for the info, we will keep an eye on it.
